src/vs/workbench/browser/layout.ts

```diff
@@ -240,6 +245,7 @@ export abstract class Layout extends Disposable implements IWorkbenchLayoutServi
 	//#endregion
 
 	private readonly parts = new Map<string, Part>();
+	private readonly auxWindows = new Map</* container */ HTMLElement, IAuxiliaryWindow>();
```

Comment:
Why not make this a `Map` of `container: HTMLElement` pointing to a `Promise` when styles have loaded.